src/network/networkd-sysctl.c

```diff
@@ -12,10 +13,25 @@
 #include "string-table.h"
 #include "sysctl-util.h"
 
-static int link_update_ipv6_sysctl(Link *link) {
+static bool common_check(Link *link, int family) {
```

Comment:
maybe call this link_is_configured_for_family()? because that's kind what this checks, no?